pyrefly/lib/alt/expr.rs

```diff
@@ -935,6 +935,19 @@ impl<'a, Ans: LookupAnswer> AnswersSolver<'a, Ans> {
                     // (https://typing.readthedocs.io/en/latest/spec/annotations.html#type-and-annotation-expressions)
                     Type::type_form(Type::type_form(targ))
                 }
+                // Handle dataclasses.InitVar[T] - should behave as just T
```

Comment:
Pyrefly already catches this issue: https://pyrefly.org/sandbox/?code=GYJw9gtgBAJghgFzgYwDZwM4YKYagSwgAcwQEoBJAO3wQDU4QAoJ62hkAbXyoQBooGBCAC6QA. I don't think this change is needed.